File: app/src/main/java/com/example/sahibj/moviebuffs/utils/DataBindingUtil.java
package com.example.sahibj.moviebuffs.utils;

import android.databinding.BindingAdapter;
import android.widget.ImageView;

import com.bumptech.glide.Glide;
import com.example.sahibj.moviebuffs.databinding.LayoutMovieItemBinding;

/**
 * Created by sahibjaspal on 1/28/18.
 */

public class DataBindingUtil {

    @BindingAdapter("android:src")
    public static void setImageUrl(ImageView imageView, String url) {
        Glide.with(imageView.getContext()).load(url).into(imageView);
    }
}

File: app/src/main/java/com/example/sahibj/moviebuffs/utils/DatabindingUtil.kt
package com.example.sahibj.moviebuffs.utils

import android.databinding.BindingAdapter
import android.widget.ImageView

import com.bumptech.glide.Glide

/**
 * Created by sahibjaspal on 1/21/18.
 */

object DatabindingUtil {

    @BindingAdapter("android:src")
    fun setImageUrl(imageView: ImageView, url: String) {
        Glide.with(imageView.context)
                .load(url)
                .into(imageView)
    }
}